components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -169,6 +172,19 @@ protected void activate(ComponentContext componentContext) throws Exception {
             bundleContext.registerService(Axis2ConfigurationContextObserver.class.getName(), tenantLoadMessageSender, null);
             KeyMgtConfigDeployer keyMgtConfigDeployer = new KeyMgtConfigDeployer();
             bundleContext.registerService(Axis2ConfigurationContextObserver.class.getName(), keyMgtConfigDeployer, null);
+
+            //Registering Notifiers
+            Notifier subscriptionsNotifier = new SubscriptionsNotifier();
```

Comment:
we can merge these two lines
bundleContext.registerService(Notifier.class.getName(), new SubscriptionsNotifier(), null);

Reply:
resolved